corehq/apps/registration/templates/registration/partials/register_new_user/submit_info.html

```diff
@@ -13,9 +13,6 @@ <h2>{% trans "Creating Account..." %}</h2>
   <p class="timeout-message" data-bind="visible: showSecondTimeout">
     <i class="fa fa-check"></i> {% trans "Created new user..." %}
   </p>
-  <p class="timeout-message" data-bind="visible: showFirstTimeout">
```

Comment:
Removing `showFirstTimeout` when there's also a `showSecondTimeout` and `showThirdTimeout` seem confusing. I'd either update the corresponding javascript or update "Created sample applications" to some other text ("Prepping application builder"?).

Reply:
Good call. I went with "Prepared project space..." as something fairly generic that matches the tenses of "Created" and "Updated" a4686a7